docs/RouteRequest.md

```diff
@@ -238,6 +239,21 @@ Penalty added for using every route that is not preferred if user set any route
 
 We return number of seconds that we are willing to wait for preferred route.
 
+<h3 id="rd_relaxTransitSearchCostCriteria">relaxTransitSearchCostCriteria</h3>
```

Comment:
`relaxTransitSearchCostCriteria` -> `relaxTransitSearchCostCriteriaAtDestination` ?

The doc need to include that setting this ONLy relaxes the pareto function at the destination, not during the search.

Reply:
Done